tests/python/unittest/test_operator.py

```diff
@@ -862,6 +862,39 @@ def fselu_grad(grad, x, y):
         check_symbolic_backward(y, [xa], [np.ones(shape)], [ga], rtol=rtol, atol=atol, dtype=dtype)
 
 
+@with_seed()
+def test_gelu():
+    CUBE_CONSTANT = 0.044715
+    TWO_OVER_PI = 0.7978845608028654
+    def g(x):
+        return TWO_OVER_PI * (x + CUBE_CONSTANT * np.power(x, 3))
```

Comment:
Often x*x*x is cheaper than x ** 3. Also, if one takes inspiration from https://en.wikipedia.org/wiki/Horner's_method , which is often more numerically stable, then g(x) could be

`return TWO_OVER_PI * x * (1 + CUBE_CONSTANT * x * x)`

Reply:
This is not the actual implementation so performance is not critical here... I'll change the internal mshadow math kernel in C++ based on the link you provided.

Reply:
Please check this part for the backend implementation: https://github.com/apache/incubator-mxnet/blob/3bf1279fcd5efb80c0d098583ae213cbd2867b66/src/operator/mshadow_op.h#L134-L147. Thanks!

Reply:
This looks correct.

Reply:
@hendrycks Thanks for your prompt reply! I'll wait for @eric-haibin-lin and @szha to give a review before we merge this in.